src/components/Proposals/ProposalBanner.js

```diff
@@ -39,43 +42,62 @@ function ProposalBanner({ onCloseBanner, ...props }) {
         >
           <IconCross
             css={`
-              color: ${theme.surfaceOpened};
+              color: ${theme.surface};
             `}
           />
         </ButtonIcon>
-
-        <h1
-          css={`
-            background: linear-gradient(to bottom, ${theme.accentStart} -50%,${
-            theme.accentEnd
-          } 60%);
-            -webkit-background-clip: text;
-            -webkit-text-fill-color: transparent;
-
-            ${textStyle('title4')}
-            font-size: ${6.5 * GU}px;
-            line-height: 1.1;
-            font-weight: 700;
-            margin-bottom: ${1.5 * GU}px;
-          `}
-        >
-          This is a test
-        </h1>
         <div
           css={`
-            ${textStyle('body1')};
-            max-width: ${95 * GU}px;
-            margin: auto;
-            margin-bottom: ${3.5 * GU}px;
-            color: ${theme.contentSecondary};
+            display: flex;
+            flex-direction: ${compactMode ? `column` : `row`};
+            align-items: center;
+            justify-content: space-around;
           `}
         >
-          Aragon Network Cash is a test organization set up as part of a new
-          round of precedence campaign disputes. Learn more on the{' '}
-          <Link href="https://aragon.org/blog/juror-dispute-guide-aragon-network-cash">
-            Aragon project blog
-          </Link>
-          .
+          <div>
```

Comment:
Should we use a `LayoutLimiter` to contain this? otherwise at wider sizes things start to expand larger than we'd like:

![image](https://user-images.githubusercontent.com/11708259/91872819-39e4a700-ec70-11ea-9e49-28a643f76210.png)

Reply:
Oh right!